src/components/Me/Navigation/Navbar.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './Navbar.css';
+
+const Navbar = () => {
+  return (
+    <div>
+      <div className="menu-tablet">
+        <div className="logo-left">
+          <img
+            class="sidenav-logo"
+            src={`${process.env.PUBLIC_URL}/codingcoach-logo-192.png`}
+            alt=""
+          />
+        </div>
+        <div className="house side-icon">
+          <img src={`${process.env.PUBLIC_URL}/images/icons/Home.png`} alt="" />
+        </div>
+        <div className="mentors side-icon">
+          <img
```

Comment:
Need to be link to the mentors list (root basically)

Reply:
Can you elaborate this @moshfeu , it's kinda unclear for me sorry.

Reply:
Sure.
It should be (or contains) `a` tag (for accessibility). Click on it should redirect the user to the app's homepage, which is the mentors list (`/` path).
Is this clear?

Reply:
For now is ok to redirect to `/`, but later on the *home* will be `/home`